declarative-examples/booleanParamBuildStep.groovy

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
The lawyers at my previous place of work would spank you, drag you by the ear to the nearest intellectual property seminar and force you to sit there for a day for asking such a question ;)